### PR TITLE
Fix #28: add script by DOM manipulation instead of `doc.write`.

### LINKED ISSUE #28: Waiting for try.kotlinlang.org
I didn't find the sequence to reproduce it but quite often the browser indicates that it is waiting for the server (even if the result seems to came back).

image

### PR DESCRIPTION
**Pull request by gzoritchak**
The issue seems to be caused by the use of `document.write`. The current PR resolves the problem by using the DOM manipulation API. Works on Chrome and Safari. Need to check on other browsers.

### CODE REVIEW

**Comment by AlexanderPrendota** [comment]
Hello! Awesome thanks for contributing. :) 
Your fix no works on IE 11. I would prefer to support IE 11. Could you test it on your side?

File: src/js-executor/index.js [hunk 1]
```diff
@@ -65,16 +65,36 @@ class JsExecutor {
     const iframe = document.createElement('iframe');
     iframe.className = "k2js-iframe";
     node.appendChild(iframe);
+    let loadedScriptCount = 0;
     this.iframe = iframe;
     let iframeDoc = this.iframe.contentDocument || this.iframe.document;
+    const head = iframeDoc.getElementsByTagName("head")[0];
+    const body = iframeDoc.getElementsByTagName("body")[0];
+    body.style = "margin: 0; overflow: hidden;";
```
Comment (AlexanderPrendota) [inline src/js-executor/index.js]: The first problem in IE11: 
should be
```js
body.style.setProperty("margin", "0")
body.style.setProperty("overflow", "hidden")
```

**Comment by AlexanderPrendota** [comment]
Also, the fix does not work in FireFox 62.0 (64-bit) =(

**Comment by gzoritchak** [comment]
I used [this workaround ](https://bugzilla.mozilla.org/show_bug.cgi?id=236547#c2). Seems ok on firefox now. 
I also changed the code for IE (but I can't test it).

**Comment by AlexanderPrendota** [comment]
If we use several Kotlin version like:

```md
<div class="kotlin-code"  data-target-platform="js">
``​`kotlin
fun main(args: Array<String>) {
    println("hi")
}
`​`​`
</div>

<div class="kotlin-code" data-target-platform="js" data-version="1.0.7">
`​`​`kotlin
fun main(args: Array<String>) {
    println("hi")
}
`​`​`
</div>


<div class="kotlin-code" data-target-platform="js" data-version="1.2.71">
`​`​`kotlin
fun main(args: Array<String>) {
    println("hi"))
}
`​`​`
</div>
```

We got three iframes:
The first one with all scripts and the others are empty.
screen shot 2018-12-26 at 11 44 32 am

**Comment by gzoritchak** [comment]
Indeed, I found a lack of symmetry and sometimes encapsulation between the different platforms.

For example `reloadIframeScripts` is called :

- from ExecutableFragment for Platform.Canvas, 
- from JsExecutor for Platform.JS.

The problem you're describing comes from the different strategies of getting the iframe node.
https://github.com/JetBrains/kotlin-playground/blob/master/src/executable-code/executable-fragment.js#L319

**Pull request closed without merging**